File: launchdarkly/ruby/src/GetFlagImportConfigurationsExample.rb
require "json"
require "launchdarkly_client"

LaunchDarklyClient.configure do |config|
    config.api_key["Authorization"] = "YOUR_API_KEY"
end

begin
    response = LaunchDarklyClient::FlagImportConfigurationsBetaApi.new.get_flag_import_configurations

    p response
rescue LaunchDarklyClient::ApiError => e
    puts "Exception when calling FlagImportConfigurationsBetaApi#get_flag_import_configurations: #{e}"
end

File: launchdarkly/python/src/GetFlagImportConfigurationsExample.py
import json
from datetime import date, datetime
from pprint import pprint

from launchdarkly_client import ApiClient, ApiException, Configuration, api, models

configuration = Configuration(
    api_key={"ApiKey": "YOUR_API_KEY"},
)

with ApiClient(configuration) as api_client:
    try:
        response = api.FlagImportConfigurationsBetaApi(api_client).get_flag_import_configurations()

        pprint(response)
    except ApiException as e:
        print("Exception when calling FlagImportConfigurationsBeta#get_flag_import_configurations: %s\n" % e)
